clients/drcov/drcov.c

```diff
@@ -156,6 +156,7 @@ options_init(client_id_t id, int argc, const char *argv[], drcovlib_options_t *o
             if (dr_sscanf(token, "%u", &verbose) != 1) {
                 USAGE_CHECK(false, "invalid -verbose number");
             }
+            ops->verbose_module_table = true;
```

Comment:
shouldn't we check the verbose value before the assignment, e.g.
ops->verbose_module_table = (verbose >= 1).
Do you just want to get the module table info without extra debug printing?